public/app/plugins/datasource/prometheus/result_transformer.ts

```diff
@@ -76,7 +76,7 @@ export class ResultTransformer {
       datapoints: dps,
       query: options.query,
       target: metricLabel,
-      labels: metricData.metric,
+      tags: metricData.metric,
```

Comment:
Is this related?  seems like something else

Reply:
Uhm, not sure, @torkel?

Reply:
Tags is the old property name in legacy format , gets translated to new DataFrame.labels

Reply:
See https://github.com/grafana/grafana/pull/18977  I moved these changes to their own PR